setup.cfg

```
@@ -48,6 +48,7 @@ spacy_architectures =
     spacy-legacy.TransitionBasedParser.v1 = spacy_legacy.architectures.parser:TransitionBasedParser_v1
 spacy_loggers =
     spacy-legacy.WandbLogger.v1 = spacy_legacy.loggers:wandb_logger_v1
+    spacy-legacy.ConsoleLogger.v1 = spacy_legacy.loggers:console_logger_v1
```

```suggestion
    spacy-legacy.ConsoleLogger.v1 = spacy_legacy.loggers:console_logger_v1
    spacy-legacy.WandbLogger.v1 = spacy_legacy.loggers:wandb_logger_v1
```
Also just to keep alphabetically